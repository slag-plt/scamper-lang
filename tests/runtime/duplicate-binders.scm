(lambda (x x y) (+ x x))

(struct foo (z y z))