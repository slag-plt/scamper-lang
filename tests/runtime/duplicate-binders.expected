:9:0: Parser error:
    Duplicate name x given in definition.
    In program: (x x y)
